app/mailers/report_mailer.rb

```diff
@@ -14,6 +14,32 @@ def budget_status
     )
   end
 
+  def annual_report(year, to_email)
+    proposals = Proposal.approved.where(client_data_type: "Ncr::WorkOrder")
+        proposals = proposals.select {|p| p.client_data.fiscal_year == year}
+        csv_string = CSV.generate do |csv|
+          csv << ["Amount", "Date Approved", "Org Code", "CL#", 
+            "Budget Activity", "SOC", "Function Code", "Building #", 
+            "Vendor", "Description", "Requestor", "Approver"]
+          for p in proposals
```

Comment:
this totally works but in Ruby we usually see `each` being used to iterate over an Array: http://ruby-doc.org/core-2.2.0/Array.html#method-i-each

so this would be:

```ruby
proposals.each do |proposal|
 ... (stuff with proposal) ...
end
```